dashboard/config/levels/custom/dance/dance_ai_code_at_measure.level

```diff
@@ -52,7 +52,7 @@
+    "authored_hints": "[{\"hint_class\":\"content\",\"hint_markdown\":\"<img src=\\\"https://images.code.org/7be0ad471681f374466d45003ac04529-image-1699980027509.png\\\" style=\\\" margin-right: 10px;  height: 20px; float: left\\\"> Connect a <code style=\\\"color:white; background-color:#566064\\\">A.I. generate effect</code> block to the <code style=\\\"color:white; background-color:#00bc3e\\\">at 2 measures</code> event in your workspace like this:\\n\\n<xml> <block xmlns=\\\"https://developers.google.com/blockly/xml\\\" type=\\\"Dancelab_atTimestampNotAfter\\\" id=\\\"S4!uAzkx{%vw@UFZX8hY\\\" deletable=\\\"false\\\" editable=\\\"false\\\">\\n  <field name=\\\"TIMESTAMP\\\">2</field>\\n  <field name=\\\"UNIT\\\">\\\"measures\\\"</field>\\n  <next>\\n    <block type=\\\"Dancelab_ai\\\" id=\\\"Ru3$z|81JlKgT/;fMb]K\\\">\\n      <field name=\\\"VALUE\\\"></field>\\n    </block>\\n  </next>\\n  </block></xml>\",\"hint_id\":\"hint1\",\"hint_type\":\"general\",\"hint_path\":\"\",\"hint_video\":\"\",\"tts_url\":\"https://tts.code.org/sharon22k/180/100/b3b7e1e2179557c6f19b8926ac224147/dance_ai_code_at_measure.mp3\"},{\"hint_class\":null,\"hint_markdown\":\"<img src=\\\"https://images.code.org/7be0ad471681f374466d45003ac04529-image-1699980027509.png\\\" style=\\\" margin-right: 10px;  height: 20px; float: left\\\"> Click on the <xml> <block type=\\\"Dancelab_ai\\\" id=\\\"Ru3$z|81JlKgT/;fMb]K\\\">\\n      <field name=\\\"VALUE\\\"></field>\\n    </block> </xml>  block to make a new effect. \",\"hint_id\":\"hint2\",\"hint_type\":\"general\",\"hint_path\":\"\",\"hint_video\":\"\",\"tts_url\":\"https://tts.code.org/sharon22k/180/100/eee95dd59098658f94fac1947af87959/dance_ai_code_at_measure.mp3\"}]",
```

Comment:
Shouldn't it be "Connect an" instead of "Connect a"?  We use "an" in the instruction for this level.